### ISSUE TITLE
 high quality rendering together with semantic labels 

### ISSUE THREAD

**Issue by johnwlambert** [facebookresearch/Replica-Dataset#41]
What would be the best way to render semantic label maps along with the highest quality Replica Dataset images, at arbitrary poses?

I'm wasn't able to reproduce the photorealistic style of the images from the paper with the Habitat Renderer (which supports semantic label map queries). Thanks very much!

My Rendering
![step_0_0_rgb](https://user-images.githubusercontent.com/16724970/69581241-2b13ea00-0fa4-11ea-997d-93880f822707.png)
![step_0_0_semantic](https://user-images.githubusercontent.com/16724970/69581243-2b13ea00-0fa4-11ea-90f6-1d39ab787635.png)

Rendering from the Paper/Repo:
![ReplicaViewer](https://user-images.githubusercontent.com/16724970/69581326-51d22080-0fa4-11ea-8b19-73c3e2b35d11.png)

**Comment by erikwijmans** [comment]
Habitat now supports ptex rendering, although I am not sure if there is anything else needed besides ptex.  CC @jstraub @bigbike

**Comment by jstraub** [comment]
Yeah habitat is the best way to achieve this. You need the converted ptex files from @bigbike.

**Comment by johnwlambert** [comment]
Thanks @jstraub, where could I find the converted ptex files @bigbike?

**Comment by bigbike** [comment]
We did not convert the ptex files. We just created some supplementary data that works with the Habitat simulator (https://github.com/facebookresearch/habitat-sim).

We pre-processed all the 18 models within the data set. The output data (1 file per model), along with a convenient setup script, were compressed into a .zip file, that could be downloaded from here: (http://dl.fbaipublicfiles.com/habitat/sorted_faces.zip)

To use it with Habitat simulator, you can:

* Download the file from the above link;
* Unzip it;
* Use the script within to copy each data file to its corresponding folder (You will have to provide the path to the folder containing all replica models. For example, ~/models/replica/);

**Comment by amir32002** [comment]
hey @johnwlambert  -- did this work out for you? I just downloaded the sorted faces and will git it a try, but I'm wondering if you ended up getting the best results through ptex rendering.

**Comment by johnwlambert** [comment]
Hey @amir32002, good to bump into you here! I haven't had a chance to try out the sorted_faces.zip file; let me know if the rendering looks better than the original files

**Comment by tsattler** [comment]
Not sure whether it is still relevant. I tried using the sorted faces, but it does not seem to make a difference in rendering quality.
